guides/release/components/helper-functions.md

```diff
@@ -302,70 +299,114 @@ Class helpers are useful when the helper logic is fairly complicated, requires
 fine-grained control of the helper lifecycle, is _stateful_ (we'll be
 discussing state in the next chapter), or requiring access to a [service](../../services/).
 
-## Built-in Helpers
+## Built-in and Common Helpers
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        Unlike nearly everything else in  template tag (`.gts`) files, these helpers do not need to be explicitly imported.
```

Comment:
these are called "keywords", and make up the fundamentals of the inner `{{ }}` language

Reply:
The links below list a weird mix of things that I would consider keywords and things I would consider "built-in helpers". Things like `textarea`, `input`, `unique-id`, `mut`, `in-element` feel like helpers to me. Things like `if`, `else`, `unless`, `yield`, `has-block` feel like keywords. 

It feels like that Ember.Templates.helpers became a dumping ground for anything that came from glimmer. These are actually the api docs that sent me down my recent foray into switching to typedoc.

Reply:
in-element and mut are also keywords. 

a _block_ `{{#in-elemnt` cannot be user-defined (without super secret knowledge of the old Curly component manager system). For consistency, I think it makes sense to to refer to if, in-element, and unless as block-keywords, most specifically.

mut isn't implementable _at all_ in user space (with any amount of secret knowledge).

Reply:
> It feels like that Ember.Templates.helpers became a dumping ground for anything that came from glimmer. 

100% agreed. it's been problematic for new learners :see_no_evil:

Reply:
it may be important to disambiguate:

- helper, modifier, component -- these are all syntax positions, and not exactly specific things
- keywords -- are things that are not imported, but can be in any of the syntax positions

- so, mut being a keyword-helper, I think is is accurate,  and clear without being too alienating